.github/workflows/report.yml

```diff
@@ -86,11 +86,26 @@ jobs:
         run: |
           bowtie info $(bowtie filter-implementations | sed 's/^/-i /') --format json > implementations.json
 
+      - name: Generate public-implementations.json file
+        run: |
+          jq 'to_entries | map({
+              (.value.source): (
+                .value + {name: .key}
+                | del(.os, .os_version, .language_version, .source)
+              )
+            })
+            | add' implementations.json > public-implementations.json
```

Comment:
This will break if/when we change the format of our own implementations.json, which is the sort of thing we're trying to avoid, so it would be slightly better to generate this from scratch via other public API commands. E.g. you could probably use `bowtie filter-implementations` + `bowtie info` I think?